### ISSUE TITLE
Allow manifest parameter names to contain the text "Exception"

### ISSUE THREAD

**Issue by aarondandy** [Azure/service-fabric-issues#97]
When defining overridable parameters in an ApplicationManifest.xml file and overriding them using Local.xml the deployment will fail with no obvious error message during a deployment if any of the paramters involved have a name containing the text "Exception". Please either correct this surprising behavior or provide a more visible description of the cause for the failed deployment.
For example
In ApplicationPackageRoot\ApplicationManifest.xml:
```
<Parameter Name="WordsWordsWords_WordsExceptionWords" DefaultValue="false" />
... 
<ConfigOverride Name="Config"><Settings><Section Name="WordsWordsWords"><Parameter Name="SomeOtherWords" Value="[WordsWordsWords_WordsExceptionWords]" />
and in ApplicationParameters\Local.xml:
<Parameter Name="WordsWordsWords_WordsExceptionWords" Value="false" />
```

https://feedback.azure.com/forums/293901-service-fabric/suggestions/15528420-allow-manifest-parameter-names-to-contain-the-text

**Comment by vturecek** [comment]
Looks like this is specific to deployments through Visual Studio using the parameter XML files. Works fine if you deploy your application directly with Service Fabric APIs (PowerShell/C#/HTTP).

A fix is coming in the Visual Studio tooling.

**Comment by vturecek** [comment]
Fixed in the 1.5 tooling release, see also: #224
